src/GlobalStyle.js

```diff
@@ -13,4 +13,11 @@ export const GlobalStyle = createGlobalStyle`
     ::before {
         box-sizing: inherit;
     }
-`;
\ No newline at end of file
+
+    body {
+        padding: 16px;
```

Comment:
Ten padding jest w celu zrobienia krawędzi dla widoku mobilnego? Jeżeli tak, to czy automatyczny margines z "Container" nie robi tego samego? 😋
Poza tym szczegółem - super!